components/filter/filter-dimension-set-value.js

```diff
@@ -9,14 +9,28 @@ class FilterDimensionSetValue extends LitElement {
 
 	static get properties() {
 		return {
+			/**
+			 * Whether the value is hidden or not (used when applying search results)
+			 */
+			hidden: { type: Boolean, reflect: true },
```

Comment:
All HTML elements get the `hidden` attribute/property, so no need to specify it here.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden

Reply:
If I don't add this though, seems like it won't show up as a `changedProperty` in `updated` (https://github.com/BrightspaceUI/core/pull/1434/files#diff-065ae76afdeb4d78f535d93bcd59d608978e7705d8c0671bf026def23ee3e641R45).  Might not matter, I'm trying out teh simplified version now.

Reply:
Ah yes, the built-in Lit property stuff. Hopefully not needed!